crates/rspack_binding_options/src/options/raw_split_chunks/raw_split_chunk_size.rs

```diff
@@ -0,0 +1,20 @@
+use derivative::Derivative;
+use napi_derive::napi;
+use rspack_plugin_split_chunks::SplitChunkSizes;
+
+#[derive(Derivative)]
+#[napi(object, object_to_js = false)]
+#[derivative(Debug)]
+pub struct RawSplitChunkSizes {
+  pub sizes: serde_json::Map<String, serde_json::Value>,
```

Comment:
why not declare `serde_json::Map<String, f64>` here.

Reply:
Using HashMap instead since Serde_json doesnt implement into_iter, debug for Map<String, (anything other than Value)>